Add info about restarting with lite members in the cluster [CTT-664] (#2038)
hazelcast/hz-docs: docs/modules/storage/pages/persistence.adoc

** 255+ characters
** Special characters (`*``:``"``'``|``<````,``>``?``/`)

> - **Lite and non-lite members**: When cluster consist of lite and non-lite members and non-lite members will be shut down before lite members, then it is not possible to restore persisted data. Lite members should be shut down before non-lite members.

Reviewer: ```suggestion
- **Lite and non-lite members**: When a cluster includes both lite and non-lite (data) members, lite members should be shut down before non-lite members. If the non-lite members are shut down first, it is not possible to restore persisted data. 
```
